tailscale/umbrel-app.yml

```diff
@@ -27,9 +27,29 @@ gallery:
 path: ""
 deterministicPassword: false
 torOnly: false
-releaseNotes: >-
-  This release updates Tailscale from v1.46.1 to v1.48.1.
+releaseNotes: |
+  This release updates Tailscale from v1.48.1 to v1.52.0 which includes: 
```

Comment:
Dude, brilliant use of  the literal `|` instead of the folded `>-` to preserve formatting. This is way better for the description and release notes.